strings/pad.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+/** FillOption Object */
+export interface FillOption {
+  /** Char to fill in */
+  char: string;
+  /** Side to fill in */
+  side: "left" | "right";
+  /** If strict, output string can't be greater than strLen*/
+  strict?: boolean;
+  /** char/string used to specify the string has been truncated */
+  strictChar?: string;
+  /** Side of truncate */
+  strictSide?: "left" | "right";
+}
+
+/**
+ * pad helper for strings. Also resolve substring
```

Comment:
It's always nice to have a bit of example code to copy and paste ....

Reply:
corrected. Sorry